src/dmd/dmangle.d

```diff
@@ -979,9 +988,9 @@ public:
             break;
         case 4:
             m = 'd';
-            for (size_t u = 0; u < e.len; u++)
+            foreach (u; 0 .. e.len)
```

Comment:
I could automatically do this replacement for all the easy cases over the entire codebase if you want to.

Reply:
Don't do it automatically. Some of those loops change the length of the loop in the loop body in unexpected ways - weird bugs result.